src/stories/2-frontend-engine/validation/validation-story-component.tsx

```diff
@@ -143,8 +143,8 @@ export const ValidationStoryComponent = ({
 							</code>
 						</pre>
 					</Section>
-				</V2_Layout.ColDiv>
-				<V2_Layout.ColDiv desktopCols={6} tabletCols={8}>
+				</Layout.ColDiv>
+				<Layout.ColDiv xlCols={6} mdCols={8}>
```

Comment:
for the next two sections, they are meant to be side-by-side on larger screens and single column on smaller screens

this can be expressed with `xxsCols={8}` and `lgCols={6}`